src/checkers/inference/util/ConstraintVerifier.java

```diff
@@ -1,15 +1,16 @@
 package checkers.inference.util;
 
 import checkers.inference.model.ConstantSlot;
+import checkers.inference.solver.backend.encoder.AbstractConstraintEncoder;
 import org.checkerframework.framework.type.QualifierHierarchy;
 import org.checkerframework.javacutil.AnnotationUtils;
 
 /**
- * This class verifies constraint with two ConstantSlots holds or not, because it
- * doesn't need solver involvment. We use this class in early verification in
- * ConstraintManager and also passes the reference to this class to solver engine
- * so that FormatTranslator can use this class to verify two-constants constraint
- * in order to encode it more efficiently.
+ * This class verifies constraint between two ConstantSlots holds or not. We use this class
```

Comment:
Avoid use personal pronoun "we". Clarify who are the potential client of this class.